### ISSUE TITLE
What if I have VPS 

### ISSUE THREAD

**Issue by JohnJadsfa** [sparkscrypto/Sparks#4]
Could you please include a list of coding if I have a vps.   I don't know how to set it up with a VPS.  Please Help

**Comment by Znuff** [comment]
https://github.com/sparkscrypto/Sparks/blob/master/How%20to%20create%20masternode.pdf

**Comment by funmbiokoya** [comment]
I am at step 7. Everytime I type “masternode outputs”, I get empty brackets as the response. No characters, just brackets. I don’t know if that’s normal or if something is wrong. 

Also, is the response I get after sending “masternode outputs” supposed to replace “collateral_output_txid” in “masternode.conf” at step 9?

**Comment by JaimeLopes** [comment]
Yes the masternode outputs result is what goes there. You need to send EXACTLY 1000.00 coins... EXACTLY!!! to your wallet to get a result. (you can send the coins to yourself by the way, assuming you have over 1000 in your wallet...

**Comment by funmbiokoya** [comment]
Thanks

**Comment by conormc11** [comment]
How do you start the syncying process of the wallet. where do you add the nodes so that it syncs the wallet?

**Comment by conormc11** [comment]
never mind i added these to the wallet config file
addnode=91.234.32.241:8890
addnode=91.201.42.245:8890
addnode=194.87.237.57:8890
addnode=178.2.46.161:8890
addnode=107.175.59.50:8890
addnode=195.181.243.145:8890
addnode=96.21.125.11:8890
addnode=45.76.252.179:8890
addnode=52.178.24.171:8890
addnode=107.172.196.13:8890
addnode=77.51.183.48:8890
addnode=185.221.152.38:8890
addnode=45.118.151.57:8890
addnode=136.144.167.226:8890
addnode=83.217.206.155:8890
addnode=82.223.43.175:8890
addnode=80.208.228.219:8890
addnode=80.208.225.163:8890
addnode=104.192.102.155:8890
addnode=209.250.235.168:8890
addnode=209.250.235.168:8890
addnode=51.15.84.213:8890
addnode=45.76.89.64:8890
addnode=80.209.230.134:8890
addnode=91.201.42.222:8890
addnode=213.136.83.147:8890
addnode=195.154.134.142:8890
addnode=163.172.68.78:8890
addnode=62.210.248.84:8890
addnode=37.215.112.178:8890
addnode=188.113.136.160:8890

**Comment by conormc11** [comment]
it is now synced

**Comment by conormc11** [comment]
Why is your discord link not working i am accessing it through masternodes online. I can not access it? please help with this.
